includes/rest-api/class-sensei-rest-api-lesson-quiz-controller.php

```diff
@@ -284,7 +284,7 @@ private function get_question_meta( array $question ) : array {
 					$meta['_question_right_answer'] = $question['answer'] ? 'true' : 'false';
 				}
 
-				if ( isset( $question['answer_feedback'] ) ) {
+				if ( isset( $question['answer_feedback'] ) || array_key_exists( 'answer_feedback', $question ) ) {
```

Comment:
I think for all these the `isset()` check is now redundant.

Reply:
Yeah it's a minor performance optimization for the case where the parameter exists but it is null. But it probably makes sense to just keep `array_key_exists`.